CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
-## 3.23.0 (Unreleased)
+## 3.24.0 (Unreleased)
+## 3.23.0 (May 26, 2020)
+
+BREAKING CHANGES:
+* The base url for the `monitoring` endpoint no longer includes the API version (previously "v3/"). If you use a `monitoring_custom_endpoint`, remove the trailing "v3/". ([#6424](https://github.com/terraform-providers/terraform-provider-google/pull/6424))
+
+FEATURES:
+* **New Data Source:** Add google_iam_testable_permissions data source ([#6382](https://github.com/terraform-providers/terraform-provider-google/pull/6382))
+* **New Resource:** `google_monitoring_dashboard` ([#6424](https://github.com/terraform-providers/terraform-provider-google/pull/6424))
+
+IMPROVEMENTS:
+
+(TODO does this belong in improvements?)
```

Comment:
Personally I don't think so, since it's not user-facing. We could stick it in a notes section?